grafana-plugin/src/models/team/team.ts

```diff
@@ -30,98 +27,11 @@ export class TeamStore extends BaseStore {
     this.currentTeam = await makeRequest('/current_team/', {});
   }
 
-  @action
-  async setCurrentTeam(teamId: Team['pk']) {
-    this.redirectingToProperTeam = true;
-
-    const team = await makeRequest(`/current_team/`, {
-      method: 'POST',
-      data: { team_id: teamId },
-    });
-
-    const pathName = getPathnameByTeamNameSlug(team.name_slug);
-
-    window.location.pathname = pathName;
-  }
-
-  @action
-  async addTeam(data: Partial<Team>) {
-    let createdTeam;
-    try {
-      createdTeam = await makeRequest('/teams/', {
-        method: 'POST',
-        data,
-      });
-    } catch (e) {
-      openErrorNotification(e.response.data);
-      return;
-    }
-
-    this.setCurrentTeam(createdTeam.pk);
-
-    Mixpanel.track('Add Team', null);
-  }
-
   @action
   async saveCurrentTeam(data: any) {
     this.currentTeam = await makeRequest('/current_team/', {
       method: 'PUT',
       data,
     });
   }
-
-  @action
-  async justSaveCurrentTeam(data: any) {
-    return await makeRequest('/current_team/', {
-      method: 'PUT',
-      data,
-    });
-  }
-
-  @action
-  async getTelegramVerificationCode(pk: Team['pk']) {
-    const response = await makeRequest(`/teams/${pk}/get_telegram_verification_code/`, {
-      withCredentials: true,
-    });
-
-    return response;
-  }
-
-  @action
-  async unlinkTelegram(pk: Team['pk']) {
-    const response = await makeRequest(`/teams/${pk}/unlink_telegram/`, {
-      method: 'POST',
-      withCredentials: true,
-    });
-
-    return response;
-  }
```

Comment:
can be removed as requests moved to https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/models/telegram_channel/telegram_channel.ts#L100

Reply:
👍